sycl/include/sycl/ext/oneapi/backend/level_zero.hpp

```diff
@@ -31,6 +31,11 @@ __SYCL_EXPORT queue make_queue(const context &Context,
 __SYCL_EXPORT queue make_queue(const context &Context, const device &Device,
                                pi_native_handle InteropHandle,
                                bool keep_ownership = false);
+__SYCL_EXPORT queue make_queue_standard_or_immediate(
```

Comment:
The other new interfaces are ending with "2" and it is not documented what it means.Can we use the same suffix for all new interfaces? And since we are going to merge them in 2024.0 maybe just use "_temporary" suffix?

Reply:
is it documented anywhere what "2" stands for and that we are aiming to remove that soon?

Reply:
I have added comments explaining the "2" versions in the places they first appear, backend.hpp and queue.hpp.